python/UniverseWrapper.cpp

```diff
@@ -473,7 +475,7 @@ namespace FreeOrionPython {
             .def("hasTag",                      &UniverseObject::HasTag)
             .add_property("meters",             make_function(ObjectMeters,                 return_internal_reference<>()))
             .def("getMeter",                    make_function(ObjectGetMeter,               return_internal_reference<>()))
-            .def("dump",                        &UniverseObject::Dump,                      return_value_policy<return_by_value>(), "Returns string with debug information, use '0' as argument.")
+            .def("dump",                        make_function(&ObjectDump,                  return_value_policy<return_by_value>()), "Returns string with debug information.")
```

Comment:
If it's a no-parameter function, it could be instead defined as a property to the API with `add_property`...?

Reply:
@Morlic-fo calls it as function.

Reply:
Is there a reason to prefer that over treating it as a porperty?

Reply:
I suppose it better to ask @Morlic-fo

Reply:
It could be defined as a property and it is ultimately a matter of taste/style.

Typically, you would use a property where it would make sense if that property was an actual member attribute of the class/object (but you need to apply some additional calculations/constraints or want to provide a read-only interface).

Having the dump-string as an actual member attribute doesn't sound very intuitive to me (and also isn't what happens on the C++ side), so a function seems to be more natural.